apps/agent/test/agent_shared.js

```diff
@@ -40,6 +41,21 @@ module.exports = (
 
     let ETH, ANY_ENTITY, APP_MANAGER_ROLE, EXECUTE_ROLE, RUN_SCRIPT_ROLE, ADD_PRESIGNED_HASH_ROLE, DESIGNATE_SIGNER_ROLE, ERC1271_INTERFACE_ID
 
+    // Error strings
+    const errors = makeErrorMappingProxy({
+      // aragonOS errors
+      APP_AUTH_FAILED: 'APP_AUTH_FAILED',
+      INIT_ALREADY_INITIALIZED: 'INIT_ALREADY_INITIALIZED',
+      INIT_NOT_INITIALIZED: 'INIT_NOT_INITIALIZED',
+      RECOVER_DISALLOWED: 'RECOVER_DISALLOWED',
+
+      // Agent errors
+      AGENT_EXEC_ETH_NO_DATA = "AGENT_EXEC_ETH_NO_DATA",
+      AGENT_EXEC_TARGET_NO_CONTRACT = "AGENT_EXEC_TARGET_NO_CONTRACT",
+      AGENT_DESIGNATED_TO_SELF = "AGENT_DESIGNATED_TO_SELF",
```

Comment:
Oops, you need to use `:` here (and in some of the other tests, where you're adding keys to an object).

Reply:
arrrg yup true, copy/paster error XD